docs/en/administration/query_profile_details.md

```diff
@@ -0,0 +1,1366 @@
+
+Description: Time taken to search the Hash Table.
+
+##### CopyRightTableChunkTime
+
+Description: Time taken to copy chunks from the right table.
+
+##### HashTableMemoryUsage
+
+Description: Memory usage of the Hash Table.
+
+##### RuntimeFilterBuildTime
+
+Description: Time taken to build runtime filters.
+
+##### RuntimeFilterNum
+
+Description: Number of runtime filters.
+
+### Window Function Operator
+
+##### ProcessMode
+
+Description: Execution mode, including two parts: the first part includes Materializing and Streaming; the second part includes Cumulative, RemovableCumulative, ByDefinition.
+
+##### ComputeTime
+
+Description: Time taken for window function calculations.
+
+##### PartitionKeys
+
+Description: Partition columns.
+
+##### AggregateFunctions
+
+Description: Aggregate functions.
+
+##### ColumnResizeTime
+
+Description: Time taken for column resizing.
+
+##### PartitionSearchTime
+
+Description: Time taken to search partition boundaries.
+
+##### PeerGroupSearchTime
+
+Description: Time taken to search Peer Group boundaries. Meaningful only when the window type is RANGE.
+
+##### PeakBufferedRows
+
+Description: Peak number of rows in the buffer.
+
+##### RemoveUnusedRowsCount
+
+Description: Number of times unused buffers are removed.
+
+##### RemoveUnusedTotalRows
+
+Description: Total number of rows removed from unused buffers.
+
+### Sort Operator
+
+##### SortKeys
+
+Description: Sorting keys.
+
+##### SortType
+
+Description: Query result sorting method: full sorting or sorting the top N results.
+
+##### MaxBufferedBytes
+
+Description: Peak size of buffered data.
+
+##### MaxBufferedRows
+
+Description: Peak number of buffered rows.
+
+##### NumSortedRuns
+
+Description: Number of sorted runs.
+
+##### BuildingTime
+
+Description: Time taken to maintain internal data structures during sorting.
+
+##### MergingTime
+
+Description: Time taken to merge sorted runs during sorting.
+
+##### SortingTime
+
+Description: Time taken for sorting.
+
+##### OutputTime
+
+Description: Time taken to build the output sorted sequence.
+
+### Merge Operator
+
+For ease of understanding various metrics, Merge can be represented as the following state machinism:
+
+```
+               ┌────────── PENDING ◄──────────┐
+               │                              │
+               │                              │
+               ├──────────────◄───────────────┤
+               │                              │
+               ▼                              │
+   INIT ──► PREPARE ──► SPLIT_CHUNK ──► FETCH_CHUNK ──► FINISHED
+               ▲
+               |
+               | one traverse from leaf to root
+               |
+               ▼
+            PROCESS
+```
+
+##### Limit
+
+- Description: Limit.
+- Level: Primary Metric
+
+##### Offset
+
+- Description: Offset.
+- Level: Primary Metric
+
+##### StreamingBatchSize
+
+- Description: Merge operates in Streaming mode, size of data processed per operation.
```

Comment:
Size of data processed per Merge operation when Merge is performed in Streaming mode